### DISCUSSION TITLE
SQL Server - dbt - Invalid object name DB_Name.information_schema.tables

### DISCUSSION THREAD

**Discussion by GeaninaTJ** [dbt-msft/dbt-sqlserver#183] [Q&A] [answered]
Hi,
After connecting to a MS SQL instance, when I tried to run the model which is a simple query (select 1 as test) I received the following error : 

Database Error
  ('42S02', "[42S02] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Invalid object name 'DB_Name.information_schema.tables'. (208) (SQLExecDirectW)")

The instance is case-sensitive.
So here comes my question : is there any way I can indicate/configure this in my project so that instead of DB_Name.information_schema.tables it would look for DB_Name.INFORMATION_SCHEMA.TABLES ?

Thanks,
Geanina

**Comment by paulcoates** [comment, marked as answer]
I've found that replacing the lower case `information_schema.tables` with `INFORMATION_SCHEMA.TABLES` in the below file fixed this for my instance that is case sensitive.

https://github.com/dbt-msft/dbt-sqlserver/blob/3d202ea4172e06233f654667ae92ab4851bcae88/dbt/include/sqlserver/macros/adapters/metadata.sql#L86

**Reply by GeaninaTJ** [reply]
Thank you, in the meantime I managed to workaround this by replacing the same in adapters.sql and catalog.sql.
